DP02_16_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<p>Don't forget to remove this before merging. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>will do after the review </p>